feature/space/src/main/java/com/yapp/itemfinder/space/addlocker/ChangeLockerImageDialog.kt

```diff
@@ -0,0 +1,45 @@
+package com.yapp.itemfinder.space.addlocker
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.setFragmentResult
+import com.yapp.itemfinder.space.databinding.ChangeLockerImageDialogBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class ChangeLockerImageDialog : DialogFragment() {
+    private var _binding: ChangeLockerImageDialogBinding? = null
```

Comment:
backing property굳이 쓸 필요 없이 하나로 쓰면 될것 같습니다!
```suggestion
    private val binding = ChangeLockerImageDialogBinding.inflate(inflater, container, false)
```

Reply:
수정했습니다~